docs/README.md

```diff
@@ -0,0 +1 @@
+View the full [Reference Documentation](https://jaeddy.github.io/workflow-execution-service-schemas/docs/) for the Workflow Execution Service API.
```

Comment:
@jaeddy -- shouldn't this point to somewhere in the GA4GH repo, not to your separate development area?

Reply:
That was vestigial from testing/demo-ing for the Monday call. Thanks for catching!